geode-core/src/main/java/org/apache/geode/admin/BackupStatus.java

```diff
@@ -19,15 +19,14 @@
 
 import org.apache.geode.cache.persistence.PersistentID;
 import org.apache.geode.distributed.DistributedMember;
+import org.apache.geode.distributed.internal.DM;
+import org.apache.geode.internal.cache.BackupUtil;
 
 /**
  * The status of a backup operation, returned by
- * {@link AdminDistributedSystem#backupAllMembers(java.io.File,java.io.File)}.
+ * {@link BackupUtil#backupAllMembers(DM, java.io.File,java.io.File)}.
  * 
  * @since GemFire 6.5
- * @deprecated as of 7.0 use the <code><a href=
- *             "{@docRoot}/org/apache/geode/management/package-summary.html">management</a></code>
```

Comment:
All the apis on the admin package are supposed to be deprecated in preference to the management apis.
Why was this deprecation tag removed?

Reply:
What we want to do is move this interface to a different package, as it is still very much in use. However, moving it there would change the public API. What is the best approach to handling this situation? Copy the interface elsewhere? Leave it deprecated, even  if it still the correct interface to use?

Reply:
Use IntelliJ refactor to move the class internal to update other classes, then recreate it where the original was so that user code doesn't break. Then change the code of the old deprecated version of the test to delegate to the newer internal version.